json-schema/json-schema/src/main/scala/endpoints4s/NumericConstraints.scala

```diff
@@ -0,0 +1,42 @@
+package endpoints4s
+
+/** Possible restrictions on the numeric value used. Needs an instance for [[Ordering]] to check whether values are
+  * valid w.r.t. to the properties.
+  * @group operations
+  */
+case class NumericConstraints[A](
```

Comment:
I know it’s a bit boring to write, but I think it would be better to not have case classes in the public algebra API. Would it be possible to make them plain classes, but to override `equals`, `hashCode`, and `toString`?

Reply:
Yes, no problem, think that would be the smart thing todo, it already paid off in the Open API model when adding these optional constraints. I'll do it after the weekend!